app/src/main/java/mozilla/lockbox/presenter/AutofillFilterPresenter.kt

```diff
@@ -7,47 +7,84 @@
 package mozilla.lockbox.presenter
 
 import io.reactivex.Observable
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.functions.Consumer
+import io.reactivex.rxkotlin.Observables
 import io.reactivex.rxkotlin.addTo
-import mozilla.appservices.logins.ServerPassword
+import kotlinx.coroutines.ExperimentalCoroutinesApi
 import mozilla.lockbox.action.AutofillAction
+import mozilla.lockbox.extensions.mapToItemViewModelList
 import mozilla.lockbox.flux.Dispatcher
 import mozilla.lockbox.flux.Presenter
+import mozilla.lockbox.model.ItemViewModel
 import mozilla.lockbox.store.DataStore
 
 interface AutofillFilterView {
-    val fillMeButtonClicks: Observable<Unit>
     val onDismiss: Observable<Unit>
-//    val filterTextEntered: Observable<CharSequence>
-//    val filterText: Consumer<in CharSequence>
-//    val cancelButtonClicks: Observable<Unit>
-//    val cancelButtonVisibility: Consumer<in Boolean>
-//    val itemSelection: Observable<ItemViewModel>
-//    fun updateItems(items: List<ItemViewModel>)
+    val filterTextEntered: Observable<CharSequence>
+    val filterText: Consumer<in CharSequence>
+    val cancelButtonClicks: Observable<Unit>
+    val cancelButtonVisibility: Consumer<in Boolean>
+    val itemSelection: Observable<ItemViewModel>
+    fun updateItems(items: List<ItemViewModel>, displayNoEntries: Boolean)
```

Comment:
I'm not wild about this method call. Either call `updateItems` or `displayNoEntries`. Currently the method is doing double duty here.